lib/connect-hazelcast.js

```diff
@@ -55,13 +55,12 @@ module.exports = (session) => {
         }
 
         this.client.getMap(this.prefix).then(map => {
-            // map.setTTL is not supported yet, so we have to deal with two operations
-            map.containsKey(sid).then(result => {
-                if (result) {
-                    return map.set(sid, session, this._extractTTL(session))
-                        .then(() => cb(null, 'OK'));
+            map.setTtl(sid, this._extractTTL(session)).then(ttlSet => {
+                if (ttlSet) {
+                    cb(null, 'OK');
```

Comment:
As I understand, we are adding these strings like "OK" for readability right? They are not used by `express-session` @huseyinbabal